docs/DeveloperGuide.md

```diff
@@ -423,8 +426,70 @@ Use Case ends.
 2. CCAConnect displays all valid commands.
 
 Use case ends.
+<br>
+
+<br>
+**Use case: UC07 - View all members**
+**Actor: User**
+**MSS**
+
+1. User requests to see all members in the contacts.
+2. CCAConnect displays all members.
+
+Use case ends.
+<br>
+
+<br>
+**Use case: UC08 - Mark attendance of members**
+**Actor: User**
+**MSS**
+
+1. User requests to mark the attendance of members.
+2. User provides the details of the session (date) to mark attendance, and list of telegrams of members to be marked.
+3. System marks the attendance of the list of members on the specified date.
+
+Use case ends.
+
+**Extensions**
+* 8a. System detects that user did not enter all the required fields.
+    * 8a1. System shows user an error message.
+    * Use case ends.
```

Comment:
i think the use case doesnt end when the input is invalid? i think either it repeats or have another step where the user re-enters the details

Reply:
I personally feels that in this case, if user input invalid commands, the state of the system will return to original state where user can input any other command (can be seen as another independent use case) if they don't want to reenter the detail for the same command again. Also other UCs in the given examples generally ends the usecase if user input incorrect message in the first step.

Reply:
ok